crates/gpio/src/backend.rs

```diff
@@ -36,8 +36,29 @@ pub(crate) enum Error {
     FailedJoiningThreads,
 }
 
+#[cfg(feature = "dummy")]
+macro_rules! gpio_after_help {
+    () => {
+        concat!(
+            "Each device number here will be used to access the corresponding /dev/gpiochipX",
+            "or simulate a GPIO device with N pins.",
```

Comment:
I think we can keep just one implementation of gpio_after_chip() and mention here:

            "or simulate a GPIO device with N pins (when mock feature is enabled).",

Reply:
ok I don't try and get cute with macros now.